src/python/pants/backend/python/util_rules/lockfile_metadata_test.py

```diff
@@ -186,60 +185,44 @@ def test_is_valid_for_v1(user_digest, expected_digest, user_ic, expected_ic, mat
 _VALID_ICS = [">=3.5"]
 _VALID_REQS = ["ansicolors==0.1.0", "requests==1.0.0"]
 
-# Different scenarios that are the same for both tool lockfiles and user lockfiles.
-_LockfileConditions = (
-    [_VALID_ICS, _VALID_ICS, _VALID_REQS, _VALID_REQS, []],
-    [_VALID_ICS, _VALID_ICS, _VALID_REQS, list(reversed(_VALID_REQS)), []],
-    [
-        _VALID_ICS,
-        _VALID_ICS,
-        _VALID_REQS,
-        [_VALID_REQS[0], "requests==2.0.0"],
-        [InvalidPythonLockfileReason.REQUIREMENTS_MISMATCH],
-    ],
-    [
-        _VALID_ICS,
-        _VALID_ICS,
-        _VALID_REQS,
-        [_VALID_REQS[0], "different"],
-        [InvalidPythonLockfileReason.REQUIREMENTS_MISMATCH],
-    ],
-    [
-        _VALID_ICS,
-        _VALID_ICS,
-        _VALID_REQS,
-        [*_VALID_REQS, "a-third-req"],
-        [InvalidPythonLockfileReason.REQUIREMENTS_MISMATCH],
-    ],
-    [
-        _VALID_ICS,
-        ["==2.7.*"],
-        _VALID_REQS,
-        _VALID_REQS,
-        [InvalidPythonLockfileReason.INTERPRETER_CONSTRAINTS_MISMATCH],
-    ],
-)
-
 
 @pytest.mark.parametrize(
-    "is_tool, lock_ics, user_ics, lock_reqs, user_reqs, expected",
-    [
-        *([True, *conditions] for conditions in _LockfileConditions),
-        *([False, *conditions] for conditions in _LockfileConditions),
-        # Tools require exact matches, whereas user lockfiles only need to subset.
-        [False, _VALID_ICS, _VALID_ICS, _VALID_REQS, [_VALID_REQS[0]], []],
+    "lock_ics, user_ics, lock_reqs, user_reqs, expected",
+    (
+        [_VALID_ICS, _VALID_ICS, _VALID_REQS, _VALID_REQS, []],
```

Comment:
The conditions in _LockfileConditions are now inlined here, since they're only used once.

Reply:
The diff is a little screwy though.